tests/ComTests.cpp

```diff
@@ -954,6 +956,7 @@ static void TestPointerConversionCombination(IFace1* p1, IFace2* p2)
 #endif
     TestSmartPointerConversion(wil::com_ptr_failfast<IFace1>(p1), wil::com_ptr_failfast<IFace2>(p2));
     TestSmartPointerConversion(wil::com_ptr_failfast<IFace1>(p1), wil::com_ptr_nothrow<IFace2>(p2));
+#endif // WIL_EXHAUSTIVE_TEST
 
 #ifdef WIL_ENABLE_EXCEPTIONS
     TestSmartPointerConversion(wil::com_ptr_nothrow<IFace1>(p1), wil::com_ptr<IFace2>(p2));

```

Comment:
>com_ptr_nothrow [](start = 36, length = 15)

In hindsight, it's probably best to switch up the type of `Ptr1`; I'll make the change